tests/common.rs

```diff
@@ -22,6 +22,7 @@ impl Factory {
             datalog: Serial::from_str("2222222222").unwrap(),
             serial: Serial::from_str("5555555555").unwrap(),
             heartbeats: None,
+            broadcast_holding_registers: None,
```

Comment:
Having this disabled by default handily keeps all the existing tests passing. I do need to add tests that exercise the `Some(true)`  case though.

Reply:
Yeah lets keep the existing behaviour for now. Its the sort of decision that could be revisited for 1.0 if we decide it'd be better turned on by default.